readthedocs/redirects/constants.py

```diff
@@ -9,11 +9,16 @@
 PAGE_REDIRECT = "page"
 EXACT_REDIRECT = "exact"
 CLEAN_URL_TO_HTML_REDIRECT = "clean_url_to_html"
+CLEAN_URL_WITHOUT_TRAILING_SLASH_TO_HTML_REDIRECT = "clean_url_without_trailing_slash_to_html"
 HTML_TO_CLEAN_URL_REDIRECT = "html_to_clean_url"
 
 TYPE_CHOICES = (
     (PAGE_REDIRECT, _("Page Redirect")),
     (EXACT_REDIRECT, _("Exact Redirect")),
     (CLEAN_URL_TO_HTML_REDIRECT, _("Clean URL to HTML (file/ to file.html)")),
+    (
+        CLEAN_URL_WITHOUT_TRAILING_SLASH_TO_HTML_REDIRECT,
+        _("Clean URL, without trailing slash, to HTML (file to file.html)"),
```

Comment:
I think that users who want this, want the other behavior (Clean URLs in this case is `/foo) -- so this feels like an even weirder default thing to support. It seems like we should just tell them how to configure their tools to output the right URLs.

Reply:
"Clean URLs" is a weird definition and I don't think there is an standard for it 😄 

In the Sphinx world it means using `/page/` instead of `page.html`. For Vercel, it means the URL won't have the extension of the filename, and it could be with or without the trailing slash. They differentiate these two things explicitly.

Reply:
Yea but it always means without the .html extension :)